code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1133,7 +1132,7 @@
 		/mob/living/simple_animal/hostile/mimic/crate,
 		/mob/living/simple_animal/hostile/mimic/crate/chest,
 		/mob/living/simple_animal/hostile/mimic/crate/item,
-		) + typesof(/mob/living/simple_animal/hostile/humanoid) + typesof(/mob/living/simple_animal/hostile/asteroid) //Exclusion list for things you don't want the reaction to create.
```

Comment:
Bruh I haven't seen a hostile core reactions in a while, usually xenobio makes friendlies and gives them to different departments